client/src/app/(modules)/practices/(main)/(details)/parsers.ts

```diff
@@ -49,7 +49,7 @@ export const getPracticeFields = (practice: Practice): FieldType[] => {
 
   if (projects && projects?.data?.length) {
```

Comment:
You need to do further changes here. The acceptance criteria says:

- The field is named “Initiative”, dynamically pluralised
- The field is located after: “Organisations” (currently “Institutions”)
- **The field’s value is either a list of links to the related projects (if it exists) or else the value of the “Project Fund” field in Strapi**

The last point means that if `projects.length` is `0`, then we display `projectName` instead (the content of the previous condition). The field “Initiative Name” would disappear.

Reply:
sorry, missed that. should be fixed now